packages/app/src/messages/errors/toggle-thread-read-status.error.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { MessagesEvent } from "App/messages/constants"
+
+export class ToggleThreadReadStatusError extends Error {
```

Comment:
```suggestion
export class ToggleThreadsReadStatusError extends Error {
```

Reply:
Fixed in [74a6af8](https://github.com/mudita/mudita-center/pull/728/commits/74a6af816f864d38862a81f9539521a7f38d27b0)